pkg/utils/progress.go

```diff
@@ -150,6 +153,7 @@ func (p *Progress) Done() {
 		}
 	}
 	p.Progress.Wait()
+	SetOutput(os.Stderr, true)
```

Comment:
It's possible to use progress bar in a mounted client, e.g: backup will call DumpMeta